server/cli/src/cli.rs

```diff
@@ -249,7 +249,7 @@ async fn main() {
                 connection_manager.clone(),
                 &hardware_id,
             ));
-            let ctx = service_provider.context().unwrap();
+            let ctx = service_provider.context("", "").unwrap();
```

Comment:
In the future we could think about a help for it, e.g. `basic_context()`

Reply:
[47ae359](https://github.com/openmsupply/open-msupply/pull/465/commits/47ae359f67d27733a7dc9aed0f23bd0227868212)